### PR TITLE
[query] store job configuration in user bucket

### PR DESCRIPTION
**Pull request by cseed**
The problem was query was writing the job configuration to the query bucket, but workers only get the user gsa, so they were unable to read the configuration.  This worked in the tests because the query and user account are both the test service account.

I can remove the query-gsa-key and the hail-query bucket after this goes in.

### CODE REVIEW

File: hail/src/main/scala/is/hail/backend/service/ServiceBackend.scala [hunk 1]
```diff
@@ -76,14 +66,15 @@ object Worker {
...
   val username: String,
   @transient val sessionID: String,
-  val billingProject: String
+  val billingProject: String,
+  val bucket: String
 ) extends BackendContext
 
 object ServiceBackend {
-  lazy val log = LogManager.getLogger("ServiceBackend")
+  lazy val log = LogManager.getLogger("is.hail.backend.service.ServiceBackend")
```
Comment (danking) [inline hail/src/main/scala/is/hail/backend/service/ServiceBackend.scala:74]: why this change?

Comment (cseed) [reply hail/src/main/scala/is/hail/backend/service/ServiceBackend.scala:74]: Consistency.  I used the fully qualified class name elsewhere.  I don't have a huge preference, although there could be name collisions otherwise.
